src/perfect_sounds_creation.py

```diff
@@ -1,32 +1,10 @@
 from music import Sound
 
-def get_perfect_sounds(sounds, beat):
-    update_beat_fractions_from_rhythmic_values(sounds)
+def get_perfect_sounds(sounds, beat_duration, beat_duration_ms):
```

Comment:
This is a little inconsistent - here beat_duration, beat_duration_ms is used to determine duration_ms of sounds, while in create_midi it is duration_ms_of_32. I think the latter is better. What do you think?